CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ## [Unreleased]
 
+## [0.0.3]
```

Comment:
remove this tag, whenever we release new version, whatever inside unreleased will be tagged automatically